modules/onmyoji/cogs/guildcmd.py

```diff
@@ -105,42 +160,80 @@ def guild_json_writeout(self):
             except Exception as e:
                 await ctx.send(f"An error occured: {e}")
-        self.guild_json_writeout()
+        self.guild_json_writeout(ctx.guild.id)
 
     @schedule_set_image.error
     async def schedule_set_image_error(self, ctx, error):
         if isinstance(error, commands.CheckFailure):
-            await ctx.send("You do not have permission to uuse this command.")
+            await ctx.send("You do not have permission to use this command.")
 
     @schedule.command(name = "message")
     @commands.check(guild_leader_check)
     async def schedule_set_message(self, ctx, *, arg=None):
         if not arg:
             try:
-                await ctx.send(f'Your current message is currently: {self.guild_info["schedule"]["message"]}')
+                await ctx.send(f"Your current message is currently: {self.guild_info[ctx.guild.id]['schedule']['message']}")
             except KeyError:
                 await ctx.send("You currently have no schedule message set. Reuse the command with a message to set one.\ne.g. `&schedule message This is a message.`")
             return
         elif "clear" in arg:
-            self.guild_info["schedule"]["message"] = None
+            self.guild_info[ctx.guild.id]["schedule"]["message"] = None
             await ctx.send("Schedule message cleared.")
         else:
             try:
-                self.guild_info["schedule"]["message"] = arg
+                self.guild_info[ctx.guild.id]["schedule"]["message"] = arg
             except KeyError:
-                self.guild_info["schedule"] = {}
-                self.guild_info["schedule"]["message"] = arg
-            await ctx.send(f"Your current schedule message is now set to: {arg}")
-        self.guild_json_writeout()
+                try:
+                    self.guild_info[ctx.guild.id]["schedule"] = {}
+                    self.guild_info[ctx.guild.id]["schedule"]["message"] = arg
+                except KeyError:
+                    self.guild_info[ctx.guild.id] = {}
+                    self.guild_info[ctx.guild.id]["schedule"] = {}
+                    self.guild_info[ctx.guild.id]["schedule"]["message"] = arg
+            await ctx.send(f"Your current schedule message is now set to: {self.guild_info[ctx.guild.id]['schedule']['message']}")
+        self.guild_json_writeout(ctx.guild.id)
 
     @schedule_set_message.error
     async def schedule_set_message_error(self, ctx, error):
         if isinstance(error, commands.CheckFailure):
             await ctx.send("You do not have permission to uuse this command.")
 
+    @commands.command(name='newmem')
+    @commands.has_permissions(manage_roles=True)
+    async def new_member(self, ctx, member=None, nickname=None):
+        new_member_channel = self.bot.get_channel(config.new_member_channel_id)
+        #new_member_role = ctx.guild.get_role(config.new_member_role_id)
+        new_member_role = ctx.guild.get_role(177994521940983809)
```

Comment:
What's this magic number from?

Reply:
I just pulled it from Discord. I shouldn't have and i should dynamically do it by way of a command, but this is a temporary stopgap for now.